.github/workflows/update-dependencies.yml

```diff
@@ -0,0 +1,28 @@
+name: Update dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 6 * * 1'  # "At 06:00 on Monday."
+
+jobs:
+  update-dependencies:
+    name: Update dependencies
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: excitedleigh/setup-nox@1.0.0
```

Comment:
I'm all for not "reinventing the wheel" but this doesn't seem very popular as of now which might raise some maintenance/availability issues. The repo is "active" and we can rework this later if this proves necessary.

Reply:
It's "official" (meaning it is listed on the nox main page), but it was broken for a long time. Originally, it forked the actions/setup-python script and modified it to run on each version of Python. This was not maintainable! About a month ago (my sense of time may be off :baby:), it was redesigned into a very simple action that uses the JS API and now should be trivial to maintain.

And, it's completely replaceable. It's identical to the following block of YAML:

```yaml
- uses: actions/setup-python@v2
  with:
    python-version: '3.6'
- uses: actions/setup-python@v2
  with:
    python-version: '3.7'
- uses: actions/setup-python@v2
  with:
    python-version: '3.8'
- uses: actions/setup-python@v2
  with:
    python-version: '3.9'
```